gulpfile.js

```diff
@@ -112,7 +112,9 @@ function createWebpackConfig(defines, output) {
       loaders: [
         {
           loader: 'babel-loader',
+          exclude: /src\/core\/(glyphlist|unicode)/, // babel is too slow
           options: {
+            presets: ['es2015'],
```

Comment:
> Agree, is there a preset for somewhat stable stuff implemented by Firefox and Chrome?

I don't know if there's a preset, but based on http://kangax.github.io/compat-table/es6/ it seems that the stable versions of both Firefox and Chrome already support most (if not all) of the ES6 features.
So, I cannot imagine that we'd run into too much trouble if we simply add a `PDFJS_NEXT` check in this patch.